docs/cosmos/apiSpec.mdx

```diff
@@ -0,0 +1,12 @@
+---
+sidebar_position: 7
+title: REST API Spec
+---
+
+import ExternalLink from '../../src/components/externals/ExternalLink';
+
+# Rest Api Spec
+
+checkout out our API endpoints [here](https://swagger.kava.io/)
```

Comment:
this text is there as a fallback incase the user disabled redirects, so that they don't end up on an empty docs page and can still navigate to the external link manually.